dojo/unittests/tools/test_auditjs_parser.py

```diff
@@ -37,19 +37,3 @@ def test_auditjs_parser_with_many_vuln_has_many_findings(self):
         self.assertEqual("connect", findings[0].component_name)
         self.assertEqual("2.6.0", findings[0].component_version)
         self.assertEqual(5.4, findings[0].cvssv3_score)
-
-
-    # Couldn't manage to generate an output with errors using AuditJS
-    # def test_auditjs_parser_empty_with_error(self):
-    #     with self.assertRaises(ValueError) as context:
-    #         testfile = open("dojo/unittests/scans/auditjs/empty_with_error.json")
-    #         parser = AuditJSParser()
-    #         findings = parser.get_findings(testfile, Test())
-    #         testfile.close()
-    #         for finding in findings:
-    #             for endpoint in finding.unsaved_endpoints:
-    #                 endpoint.clean()
-    #         self.assertTrue(
-    #             "AuditJS report contains errors:" in str(context.exception)
-    #         )
-    #         self.assertTrue("ECONNREFUSED" in str(context.exception))
```

Comment:
For this one you could provide a file which is not JSON.

Reply:
OK, I'll do that